docs/reference/UDS Core/functional-layers.md

```diff
@@ -46,3 +46,37 @@ packages:
     repository: ghcr.io/defenseunicorns/packages/uds/core-backup-restore
     ref: 0.29.0-upstream
 ```
+
+## Layer Selection
+
+Layer selection will always be deployment-specific but below are guidelines for what layers to consider for your deployment.  The *required* layer guidance follows the [Big Bang Conformant Stack](https://repo1.dso.mil/big-bang/product/bbtoc/-/blob/master/policy/conformance.md?ref_type=heads) though if you are not bound by that document you may need to tweak them for your needs - UDS Core Base is the only *technical* layer required to install most UDS Packages.
+
+| Layer | Selection Criteria |
+|-------|--------------------|
+| **UDS Add-ons*** |
```

Comment:
TBD on naming here, I think add-ons makes sense but not sure if it might conflict with other things we are calling add-ons? Cc @austenbryan